internal/pkg/poster/poster.go

```diff
@@ -185,15 +185,14 @@ func (p *Poster) sendPoSt(ctx context.Context, stateView *appstate.View, candida
 	poStCandidates := make([]abi.PoStCandidate, len(candidates))
 	for i, candidate := range candidates {
 		poStCandidates[i] = abi.PoStCandidate{
-			RegisteredProof: abi.RegisteredProof_WinStackedDRG32GiBPoSt,
+			RegisteredProof: abi.RegisteredProof_StackedDRG32GiBPoSt,
```

Comment:
:warning: WinStackedDRG was removed from specs actors, is this the correct substitution?

Reply:
@anorth ?

Reply:
received verbal confirmation this is the correct value.